themes/bootstrap3/js/hold.js

```diff
@@ -34,11 +34,20 @@ function setUpHoldRequestForm(recordId) {
           $.each(response.data.locations, function holdPickupLocationEach() {
             var option = $('<option/>').attr('value', this.locationID).text(this.locationDisplay);
             // Make sure to compare locationID and defaultValue as Strings since locationID may be an integer
-            if (String(this.locationID) === String(defaultValue) || (defaultValue === '' && this.isDefault && $emptyOption.length === 0)) {
+            if (String(this.locationID) === String(defaultValue) ||
+              (defaultValue === '' && this.isDefault && $emptyOption.length === 0) ||
+              (response.data.locations.length == 1)) {
```

Comment:
This is a slight inconsistency with what happens without request groups, as in that case, the single non-hidden location option is not actually selected.  But it seems necessary to change the active value in the drop-down, and I don't think there is any harm in it.